k8s/filings-ui.dc-srvc-rt-cmap-kcmap.param

```diff
@@ -21,14 +21,14 @@ MOUNT_PATH="/var/www/html/business/config"
 DOMAIN="dev.bcregistry.ca"
 PATH="/business"
 # For config maps
-BUSINESSES_URL="https://dev.bcregistry.ca/business/auth/"
+BUSINESSES_URL="https://dev.bcregistry.ca/business"
 AUTH_URL="https://dev.bcregistry.ca/business/auth/"
 LEGAL_API_URL="https://legal-api-dev.pathfinder.gov.bc.ca/api/v1/"
 AUTH_API_URL="https://auth-api-dev.pathfinder.gov.bc.ca/api/v1/"
 CREATE_URL= "https://business-create-dev.pathfinder.gov.bc.ca/businesses/create/"
 EDIT_URL= "https://business-create-dev.pathfinder.gov.bc.ca/businesses/edit/"
 PAY_API_URL="https://pay-api-dev.pathfinder.gov.bc.ca/api/v1/"
-KEYCLOAK_CONFIG_PATH="https://dev.bcregistry.ca/business/auth/config/kc/keycloak.json"
+KEYCLOAK_CONFIG_PATH="https://dev.bcregistry.ca/business/config/kc/keycloak.json"
```

Comment:
Good change. We were sharing the auth keycloack.json file but now we will use our own :)